app/src/main/java/com/zulip/android/activities/ZulipActivity.java

```diff
@@ -1113,11 +1113,17 @@ private void sendMessage() {
                 return;
             } else {
                 try {
-                    Cursor streamCursor = makeStreamCursor(streamActv.getText().toString());
-                    if (streamCursor.getCount() == 0) {
+                    //check whether stream exists or not
+                    if (!isStreamExist(streamActv.getText().toString())) {
```

Comment:
But for stream does not exist -> We are showing error in editText
and for stream not subscribed -> We are showing toast 
In both cases messages are different. How to handle that?